x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/indexlifecycle/AbstractStepTestCase.java

```diff
@@ -26,4 +26,12 @@ public void testHashcodeAndEquals() {
     public static StepKey randomStepKey() {
         return new StepKey(randomAlphaOfLength(10), randomAlphaOfLength(10), randomAlphaOfLength(10));
     }
+
+    public void testStepNameNotError() {
```

Comment:
Is your motivation for this that the random key might randomly choose `ERROR` as the step key?

Reply:
correct. doesn't seem to be the case now since almost everyone is random string of length 10, but it has bit me before, so I thought it would help to be extra clear